src/NuGet.Clients/NuGet.VisualStudio.Interop/NuGet.VisualStudio.Interop.sln

```diff
@@ -0,0 +1,27 @@
+﻿Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
What is this new file? Are we adding a new solution?

Reply:
By accident added this one.